test/mongoose_graphql_SUITE.erl

```diff
@@ -1141,16 +1244,27 @@ meck_module_and_service_checking(Config) ->
     LoadedModules = #{<<"test-domain.com">> => [mod_a, mod_d],
                       <<"localhost">> => [mod_a, mod_b, mod_c]},
     LoadedServices = [service_a, service_b],
+    LoadedDBs = [db_a],
     % gen_mod
     meck:new(gen_mod, [no_link]),
     meck:expect(gen_mod, is_loaded,
                 fun (Domain, M) -> lists:member(M, maps:get(Domain, LoadedModules, [])) end),
     % mongoose_service
     meck:new(mongoose_service, [no_link]),
     meck:expect(mongoose_service, is_loaded, fun (M) -> lists:member(M, LoadedServices) end),
+    % mongoose_config
+    meck:new(mongoose_config, [no_link]),
```

Comment:
We tend to just use `mongoose_config` instead of mocking it. It is a simple persistent term, so I don't think mocking is needed. Also, when changing `get_opt` to `lookup` it would break. See other small tests for example uses of `mongoose_config`.